### PR TITLE
Moves the frontend generic configuration to a new plugin

### PR DESCRIPTION
**Pull request by hbcarlos**
cc @davidbrochart 

This PR moves the generic frontend configuration to a new plugin.

The idea is that another fps-base app could reuse the kernel plugin without depending on the fps-lab plugin, being able to implement a new frontend that reuses the kernel plugin.

### CODE REVIEW

**Comment by hbcarlos** [comment]
I believe the Check Release workflow is failing because of an unrelated issue. It is failing in the main branch too, and I think it is related to the order used to install the packages in the step `check-python`.

File: pyproject.toml [hunk 1]
```diff
@@ -44,7 +44,7 @@ skip = [ "check-links", "check-manifest",]
 max-line-length = 100
 
 [tool.jupyter-releaser.options]
-python_packages = [ "plugins/auth:fps-auth", "plugins/auth_fief:fps-auth-fief", "plugins/contents:fps-contents", "plugins/jupyterlab:fps-jupyterlab", "plugins/kernels:fps-kernels", "plugins/lab:fps-lab", "plugins/nbconvert:fps-nbconvert", "plugins/retrolab:fps-retrolab", "plugins/terminals:fps-terminals", "plugins/yjs:fps-yjs", "plugins/login:fps-login", ".:jupyverse:fps-auth,fps-auth-fief,fps-contents,fps-jupyterlab,fps-kernels,fps-lab,fps-nbconvert,fps-retrolab,fps-terminals,fps-yjs",]
+python_packages = [ "plugins/auth:fps-auth", "plugins/auth_fief:fps-auth-fief", "plugins/contents:fps-contents", "plugins/frontend:fps-frontend", "plugins/jupyterlab:fps-jupyterlab", "plugins/kernels:fps-kernels", "plugins/lab:fps-lab", "plugins/nbconvert:fps-nbconvert", "plugins/retrolab:fps-retrolab", "plugins/terminals:fps-terminals", "plugins/yjs:fps-yjs", "plugins/login:fps-login", ".:jupyverse:fps-auth,fps-auth-fief,fps-contents,fps-jupyterlab,fps-kernels,fps-lab,fps-nbconvert,fps-retrolab,fps-terminals,fps-yjs",]
```
Comment (davidbrochart) [inline pyproject.toml:47]:
```suggestion
python_packages = [ "plugins/auth:fps-auth", "plugins/auth_fief:fps-auth-fief", "plugins/contents:fps-contents", "plugins/frontend:fps-frontend", "plugins/jupyterlab:fps-jupyterlab", "plugins/kernels:fps-kernels", "plugins/lab:fps-lab", "plugins/nbconvert:fps-nbconvert", "plugins/retrolab:fps-retrolab", "plugins/terminals:fps-terminals", "plugins/yjs:fps-yjs", "plugins/login:fps-login", ".:jupyverse:fps-auth,fps-auth-fief,fps-contents,fps-jupyterlab,fps-kernels,fps-lab,fps-frontend,fps-nbconvert,fps-retrolab,fps-terminals,fps-yjs",]
```

**Review by davidbrochart — CHANGES_REQUESTED** [review]
Thanks Carlos!
The Check Release workflow is failing because we have not published `fps-frontent` to PyPI yet, so that's expected. I'll do it after merging this PR.
@blink1073 mentioned that jupyter-releaser should support local packages though, but I don't know if there's something to do to enable that?

**Comment by hbcarlos** [comment]
Thanks, @davidbrochart! I pushed a commit with the changes.

**Review by davidbrochart — APPROVED** [review]

**Comment by blink1073** [comment]
> local packages

It should already be doing that in check release, using a local pypi server, but that server only serves files that you've published.

**Pull request merged**